radio/src/mixer.cpp

```diff
@@ -534,13 +534,6 @@ void evalInputs(uint8_t mode)
     if (v < -RESX) v = -RESX;
     if (v >  RESX) v =  RESX;
 
-#if defined(PCBTARANIS) && !defined(PCBX7) && !defined(SIMU)
-    // TODO why not in the driver?
```

Comment:
@3djc I have removed this `TODO`, I have only checked X9D+, would you check X9E?

Reply:
Is this in the driver now?  There's an exception somewhere in Simulator/simu for this also... have to find it.